analyzers/tests/SonarAnalyzer.Test/TestCases/SpecifyRouteAttribute.cs

```diff
@@ -0,0 +1,111 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Routing;
+
+using MA = Microsoft.AspNetCore;
+
+public class RouteTemplateIsNotSpecified : Controller
```

Comment:
Initial thread on `rspec`: https://github.com/SonarSource/rspec/pull/3676/files#r1502696074

The new name is good. I would just add the `Controller` suffix to all controllers, to have something that would actually work in ASP.NET (without adding special attributes such as `[Controller]`) and could be tested by just copy-pasting into a valid ASP.NET project. We tried to consistently do the same in S6930.